src/components/Search.js

```diff
@@ -19,12 +19,39 @@ const SearchOfflinePanel = () => {
   )
 }
 
+const ConsentPopup = ({ close, i18n }) => {
+  const onKeyRight = () => {
```

Comment:
`onKeyRight` and `onKeyLeft` are basically identical except for one value. You can explore using a factory function to build handlers preconfigured for a specific response. Something like:
```
const handleConsent = response => {
  return () => {
    setConsentStatus(response)
    setConsentConfirmationStatus(true)
    close()
  }
}
...
onKeyRight: handleConsent(true),
onKeyLeft: handleConsent(false),
```